### PR TITLE
Misconfigured DNS

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/misconfigured_dns/template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Misconfigured DNS
+
+## Overview of the Vulnerability
+
+Domain Name System (DNS) misconfigurations can result in internal information being leaked publicly. The DNS misconfiguration identified on the endpoints allows an attacker to view the internal network structure of the domain, transfer the zone file from one DNS to . From here, the attacker can use this information to form other attacks on the network and its users.
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/misconfigured_dns/template.md:5]: Unsure of this wording and what it means
> transfer the zone file from one DNS to .

**Review by evildaemond — CHANGES_REQUESTED** [review]
Need to update wording before we introduce this into prod

File: submissions/description/server_security_misconfiguration/misconfigured_dns/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/template.md]: Thanks @evildaemond, had a D'oh moment writing that sentence...

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**